### PR TITLE
fix(valid-describe): add check for empty arguments

### LINKED ISSUE #93: valid-describe throws an error with empty arguments
`describe()` causes an error when checking `node.arguments[0]` attributes.

### PR DESCRIPTION
**Pull request by lionize**
This PR fixes #93.

### CODE REVIEW

File: rules/__tests__/valid-describe.test.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const ruleTester = new RuleTester({
 
 ruleTester.run('valid-describe', rule, {
   valid: [
+    'describe()',
```
Comment (SimenB) [inline rules/__tests__/valid-describe.test.js]: An empty `describe` isn't valid - could you make it be a linting error instead?

**Comment by mchandler-cng** [comment]
@SimenB Sorry about that, ended up rather busy. Thanks for following through!

**Comment by SimenB** [comment]
No worries 🙂 Seemed like a pretty critical error to land, and your PR was basically there

**Pull request merged**